app/Console/Commands/FetchNativeBalances.php

```diff
@@ -56,18 +57,22 @@ private function handleAllWallets(Collection $networks): void
         $onlyOnline = $this->option('only-online');
 
         Wallet::query()
+            ->select(['id', 'address'])
             ->when($onlyOnline, fn ($query) => $query->online())
             ->when(! $onlyOnline, fn ($query) => $query->recentlyActive())
             ->chunkById(100, function ($wallets) use ($networks) {
-                $wallets->each(fn ($wallet) => $this->handleWallet($wallet, $networks));
+                $this->process($wallets, $networks);
```

Comment:
We process wallets in chunks of `100`, while the Moralis endpoint only accepts up to `25` addresses.

This will probably stop working once we exceed the threshold (unless I missed something). So maybe we just chunk by 25 here?

https://docs.moralis.io/web3-data-api/evm/reference/get-native-balances-for-addresses?chain=eth&wallet_addresses=[]

![image](https://github.com/ArdentHQ/dashbrd/assets/120286271/19cf0f29-fcbc-4a43-90e3-bbcd65074541)

Reply:
Ah, right! Nice catch. Should be fine now.